crates/sui/tests/cli_tests.rs

```diff
@@ -326,6 +326,31 @@ async fn test_ptb_publish_and_complex_arg_resolution() -> Result<(), anyhow::Err
     Ok(())
 }
 
+#[sim_test]
+async fn test_ptb_publish() -> Result<(), anyhow::Error> {
```

Comment:
It turns out the test above this one uses the client command `::Publish` because it needs to access the transaction results, and so doesn't actually exercise the publish via PTBs (which this test now does).